projects/schematics/src/migrations/4_0/removed-public-api-deprecations/removed-public-api-deprecation.ts

```diff
@@ -250,253 +167,6 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
-    comment: `'${SYNCED_ASM_STATE}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/services/asm-state-persistence.service.ts
-  {
-    node: ASM_STATE_PERSISTENCE_SERVICE,
-    importPath: SPARTACUS_CORE,
-    comment: `'${ASM_STATE_PERSISTENCE_SERVICE}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/asm-ui.action.ts
-  {
-    node: ASM_UI_UPDATE,
-    importPath: SPARTACUS_CORE,
-    comment: `'${ASM_UI_UPDATE}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/asm-ui.action.ts
-  {
-    node: ASM_UI_UPDATE_CLASS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${ASM_UI_UPDATE_CLASS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/asm-ui.action.ts
-  {
-    node: ASM_UI_ACTION,
-    importPath: SPARTACUS_CORE,
-    comment: `'${ASM_UI_ACTION}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_CLASS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_CLASS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_FAIL,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_FAIL}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_FAIL_CLASS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_FAIL_CLASS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_SUCCESS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_SUCCESS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_SUCCESS_CLASS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_SUCCESS_CLASS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_RESET,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_RESET}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_SEARCH_RESET_CLASS,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_SEARCH_RESET_CLASS}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/customer.action.ts
-  {
-    node: CUSTOMER_ACTION,
-    importPath: SPARTACUS_CORE,
-    comment: `'${CUSTOMER_ACTION}' was moved to @spartacus/asm/core.`,
-  },
-  // projects/core/src/asm/store/actions/logout-agent.action.ts
-  {
```

Comment:
Now I realized that all these migrations were not working, as all the actions are grouped under `AsmActions`